src/shared/components/matrix/myProfileForm/myProfileForm.styles.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import { ReactComponent as ArrowRight } from '../../../../images/icons/arrow-right.svg';
+
+export const ButtonsContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  gap: 40px;
+`;
+
+export const ArrowRightIcon = styled(ArrowRight)`
```

Comment:
and then you can also remove this from here

Reply:
✅